src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -653,10 +671,12 @@ public bool EnableRaisingEvents
         {
             get
             {
+                CheckDisposed();
```

Comment:
Same here, etc etc

Reply:
So, where I should leave that call? Because I follow this comment
https://github.com/dotnet/runtime/issues/45149#issuecomment-733044778

Reply:
Hi @eiriktsarpalis, any updates?

Reply:
> So, where I should leave that call?

It should be removed altogether from the particular property. Per https://github.com/dotnet/runtime/pull/46593/files/46bb8fef77a4653e79b1179f45fd20d290e7aed1#r552128870 throwing exceptions in properties that used to work is a breaking change. The scope of #45149 concerns cases where we throw `InvalidOperationException` instead of `ObjectDisposedException`.

Reply:
FYI @adamsitnik @Jozkee @carlossanlop

Reply:
@eiriktsarpalis please review if I remove from all proper places and let me know when I should remove others. Thx

Reply:
Changes look good, though I defer final decision to @adamsitnik @Jozkee and @carlossanlop on final decision since they are the new Process are owners.